### ISSUE TITLE
[Github Actions] Installation with micromamba fails on windows-latest

### ISSUE THREAD

**Issue by dnerini** [pySTEPS/pysteps#265]
Recent CI/CD runs fail on windows when micromamba tries to install the environment.

~~This could be related to the [recent change to Windows Server 2022](https://github.blog/changelog/2022-01-11-github-actions-jobs-running-on-windows-latest-are-now-running-on-windows-server-2022/) for all windows-latest github actions.~~

Investigating ...

**Comment by dnerini** [comment]
seems to be caused by the pip installation of cookiecutter:
```
  Installing pip packages: cookiecutter
  'DOSKEY' is not recognized as an internal or external command,
  operable program or batch file.
  'chcp' is not recognized as an internal or external command,
  operable program or batch file.
  'chcp' is not recognized as an internal or external command,
  operable program or batch file.
  info     ****************** Backtrace Start ******************
  debug    Loading configuration
  trace    Compute configurable 'create_base'
  trace    Compute configurable 'no_env'
  trace    Compute configurable 'no_rc'
  trace    Compute configurable 'rc_files'
  trace    Compute configurable 'root_prefix'
  trace    Get RC files configuration from locations up to HomeDir
  critical Cannot determine HOME (checked USERPROFILE, HOMEDRIVE and HOMEPATH env vars)
  info     ****************** Backtrace End ********************
  critical pip failed to install packages
  Error: The process 'C:\Windows\System32\WindowsPowerShell\v1.0\powershell.exe' failed with exit code 1
  C:\Windows\System32\WindowsPowerShell\v1.0\powershell.exe -command "if (!(Test-Path $profile))
```

**Comment by aperezhortal** [comment]
The issue was the `pip` call from the version of micromamba-bin used by the action (0.21). Specifying the mamba version to 0.20 seems to solve the issue:
```
- name: Install mamba and create environment
  uses: mamba-org/provision-with-micromamba@main
  with:
    micromamba-version: 0.20.0     # This version works, 0.21 does not.   
    environment-file: ci/ci_test_env.yml
    environment-name: test_environment
    extra-specs: python=${{ matrix.python-version }}
```

I will try to create a minimal example reproducing the error and open an issue in the mamba repository.

**Comment by dnerini** [comment]
I found this issue on their repo: https://github.com/mamba-org/mamba/issues/1468 could that be it?
The fix was released with 0.21.1 few hours ago...

edit: just tested with micromamba 0.21.1 and [it works](https://github.com/pySTEPS/pysteps/actions/runs/1834372733) all good. So looks like the issue was fixed with the latest patch of mamba and no more action is required on our side...
